site/static/index.html

```diff
@@ -154,8 +154,6 @@
                         enabled: false,
```

Comment:
So we'll want to remove this logic here since we're no longer consistent across all three graphs as to the maximum.